add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -556,6 +556,7 @@ var LoopRoomsInternal = {
       for (let room of roomsList) {
         // See if we already have this room in our cache.
         let orig = this.rooms.get(room.roomToken);
+        // let orig = extend({}, this.rooms.get(room.roomToken));
```

Comment:
We've just discussed this and determined that the copy of the room stored in `this.rooms` is not modified. So we don't need the `extend` function here.